### PR TITLE
using Dialog mixins for alerts

### PR DESCRIPTION
**Pull request by knowbody**
replaced the `alert()` in `flashAlert()` with the `Dialogs` mixin, previous was causing the app to crash (on clicking for example "Primary Button").

### CODE REVIEW

File: src/js/views/details.js [hunk 1]
```diff
@@ -47,11 +48,13 @@ module.exports = React.createClass({
 	},
 
 	flashAlert: function(alertContent) {
-		alert(alertContent);
+		return this.showAlertDialog({ message: alertContent }, function() {
+			this.showView('home', 'fade')
```
Comment (dcousens) [inline src/js/views/details.js]: This shouldn't be here.

**Comment by dcousens** [comment]
Remove the default `showView` action and instead maybe do: `function(alertContent, callback) {` and , `callback || function() {}`

**Comment by knowbody** [comment]
have a look now

**Comment by JedWatson** [comment]
Thanks @knowbody, sorry for the delay merging too.

**Comment by knowbody** [comment]
no problem :) @JedWatson will contribute more in a few weeks after uni is done

**Pull request merged**